scripts/build-crossdock.sh

```diff
@@ -11,14 +11,19 @@ make build-and-run-crossdock
 if [[ "$BRANCH" == "main" ]]; then
   echo 'upload images to dockerhub/quay.io'
   REPO=jaegertracing/test-driver
-  IFS=" " read -r -a IMAGE_TAGS <<< "$(bash scripts/compute-tags.sh ${REPO})"
-  IMAGE_TAGS+=("--tag" "docker.io/${REPO}:${COMMIT}" "--tag" "quay.io/${REPO}:${COMMIT}")
+
+  TARGET_ARCH="linux/arm64"
+
+  IMAGE_TAGS=("--tag" "docker.io/${REPO}:${COMMIT}-${TARGET_ARCH}" "--tag" "quay.io/${REPO}:${COMMIT}-${TARGET_ARCH}")
```

Comment:
we do not include arch in the image tags. You can see in the production images that the same tag could represent multiple architectures, e.g.

<img width="1872" alt="image" src="https://github.com/jaegertracing/jaeger/assets/3523016/c3570803-0cc0-4592-9a8a-7649d4c4fdf4">

Reply:
Yes, thanks for clarifying. Tried a different approach now, went through the QEMU documentation, and noticed we can specify platforms as an input to QEMU, if we want only specific platforms to be built? Is this approach right?